cdap-ui/app/cdap/components/ManagementView/index.js

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, {Component, PropTypes} from 'react';
+require('./ManagementView.less');
+var classNames = require('classnames');
+import InfoCard from '../InfoCard/index.js';
+import ServiceStatusComponent from '../ServiceStatusComponent/index.js';
+import ServiceStatusPanel from '../ServiceStatusPanel/index.js';
+import AdminDetailPanel from '../AdminDetailPanel/index.js';
+import ConfigurePane from '../ConfigurePane/index.js';
+import OverviewPane from '../OverviewPane/index.js';
+
+const propTypes = {
+  data: PropTypes.object
+};
+
+class ManagementView extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      application: 'CDAP',
+      lastUpdated: 15,
+      navUnderlineClass: 'one',
+      loading: true
+    };
+    this.interval = undefined;
+    this.setToCDAP = this.setToCDAP.bind(this);
+    this.setToYARN = this.setToYARN.bind(this);
+    this.setToHBASE = this.setToHBASE.bind(this);
+  }
+
+  setToCDAP() {
```

Comment:
Do we need 3 functions for this? Can we have something like,
`setToContext('cdap')` and have the state use the function parameter?